CenterForOpenScience/osf.io
# Node taxonomies backend [OSF-8275]

## Purpose
Add Subjects to nodes and registrations

## Changes
* Add Subjects to nodes and registrations
* Add tests

## Side effects
None expected

## QA Notes
* No UI changes.
* API change - nodes now accept `subjects`, using the same conventions as preprints. This means:
    * If you're an admin, you can set the subjects on the node.
    * If you're not an admin, you can't.
    * If you have view permissions on the node, you can see what subjects it has.

## Ticket
[[OSF-8275]](https://openscience.atlassian.net/browse/OSF-8275)
